src/js/containers/agency/visualizations/ObligatedContainer.jsx

```diff
@@ -64,7 +65,8 @@ export class ObligatedContainer extends React.PureComponent {
 
                 this.setState({
```

Comment:
It worries me that this doesn't break our tests. This indicates this is either not tested (so we should add tests) or it's not testing the right thing (ie we should fix it)...

Reply:
Added a new test that checks that the state variables are set correctly, including outlay amount